### PR TITLE
dom-event-delegation - Ashelle Cancio

### PR DESCRIPTION
**Pull request by arcan9**
js-dom-delegation1
js-dom-delegation2

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  var closestTaskList = event.target.closest('.task-list-item');
+  console.log('closest .task-list-item', closestTaskList);
+  closestTaskList.remove();
+
+  if (event.target.tagName !== 'BUTTON') {
+    // eslint-disable-next-line no-useless-return
+    return;
+  }
```
Comment (Zhiwutian) [inline dom-event-delegation/main.js:15]: make sure you perform this check before the closest element is found

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
very close, one small update!

File: dom-event-delegation/main.js (hunk 1 above)
Comment (arcan9) [reply dom-event-delegation/main.js:15]: ah, that's what I was going to ask about. lol. Let me re-check

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**